src/bin/stratis_dumpmetadata.rs

```diff
@@ -33,5 +34,8 @@ fn main() {
     }
     let devpath = args[1].clone();
 
-    run(devpath);
+    match run(devpath) {
+        Ok(()) => {}
+        Err(_e) => println!("Error encountered"),
```

Comment:
You might as well use ```eprintln!``` here so the text goes out on stderr.

Reply:
Right here is where you get to decide what the exit code should be be as well. In our simple model, we can just have the usual two exit codes, and refine it as we might need later.

I recall you were using subprocess::exit in an example; that's the way to go.

Reply:
I think you mean "process::exit"; "subprocess" might be from Python.